assets/data/trip01.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
It seems tha this file can be removed.

Reply:
Yes, it was a remnant from a different solution model. Thanks.